src/cli.js

```diff
@@ -236,6 +236,17 @@ Options
                 reporter.isReportServer = boolValue();
                 break;
 
+            case "report-root":
+                if(!value.match((/^(.*?)[\\\/]([^\\\/]+[a-zA-Z])$/) || (/^[^\/\: ]+(\:[0-9]+)?$/))) {
```

Comment:
You probably didn't want to put whitespace at the sides of the `||`
Also, `[a-zA-Z]` might be a but restrictive, as file/dir names can contain other characters

I haven't really looked into it yet, but this regex seems a little restrictive.
Also, would we want to use a domain or domain:port as the report path? We're talking a file in the filesystem here, right?

Reply:
Yeah i can look for change my regex, but for domaine, domaine:port its for AWS, Azur location in my company we save the files in an Amazon system (not local)

Reply:
What do you think of as regex more permissive for the path?

Reply:
Maybe allow any string, and if it's a bad path an exception will get thrown somewhere?